how-to/bootstrap_permissions.rst

```diff
@@ -0,0 +1,81 @@
+Bootstrap Admin Permissions
```

Comment:
I think the "bootstrap" in title is a bit misleading, because we're also covering the setting of user permissions. What do you think?

Reply:
agreed.. perhaps `Permission management`?